maintainers/maintainer-list.nix

```diff
@@ -10299,6 +10299,12 @@
     githubId = 2212422;
     name = "uwap";
   };
+  V = {
```

Comment:
Can you please use deviant?

Reply:
No.

Reply:
In my opinion a friendlier reply and a reason for your refusal would have been more appropriate. @SuperSandro2000 does an incredible job as a reviewer and I agree with him, a maintainer with a one letter nick name is harder to find and to track down than maintainers with longer names.

Reply:
We have the `github` and `githubId` fields for GitHub usernames. For the sake of all that is holy, don't bother people about their names.
If you want to pick a fight over that, you're welcome to do that with me instead.

Reply:
@dschrempf

a) You're lacking context. Sandro has previously been petty and rude when reviewing PRs I've made. Please don't jump in and assume things. While I have him blocked, GitHub doesn't actually do anything other than show this to him, filter notification e-mails to me, and hide random pieces of the UI. Since Sandro hasn't taken the hint by now that *maybe* it's impolite to continue harassing someone who has asked them to stop doing so, I think actually giving him a response at all is being more courteous than is necessary.
b) I'm V on IRC. It's really not hard to find me.
c) It's trivial to find my e-mail address from the commit log.
d) If you'd like to convince @v to transfer their handle to me, or convince me to change my IRL name to 'deviant' and handle all the bureaucratic overhead, you're more than welcome to. I'm currently quite happy with my name as-is, though, and don't care about consistency across handles.
e) The list contains GitHub handles and IDs for a reason. The identifiers are simply that: names. V is who I am, not what I am.

I hope you find this explanation satisfactory.

Reply:
@edef1c Thanks for this information. I expressed my opinion in a nice way and by no means wanted to pick a fight. This was solely your interpretation. I was complaining about the briefness and about the lack of friendliness of @deviant towards @SuperSandro2000.

@deviant I didn't know that @SuperSandro2000 has been unfriendly to you about your pull requests, I am sorry. The accusation "harassment" is a very harsh one, though, you may want to think it over. Anyways, how should I have known? I feel like I jumped into a fight the three of you picked some time ago :smile:. From my point of view it was you being unfriendly. Maybe the two/three of you should talk this over in this case? Thanks for your explanation.

I am available at #nixos now, and plan to be more active there in the future, if you want to continue this conversation.

Reply:
@dschrempf I'm not interested in having this conversation, sorry. Please try to avoid engaging in backseat moderation/tone policing in future. [Willfully ignoring](https://github.com/NixOS/nixpkgs/pull/118712#issuecomment-817898639) being blocked by *multiple people* and choosing to continue to interact with them regardless is harassment *by definition*, not a "harsh" accusation. You didn't *need* to know, as we'd not be conversing presently had you not chosen to jump in earlier. If you think people haven't *repeatedly* tried to talk to him about his behaviour, you're sorely mistaken. There's no fight desired here, just someone lacking in social skills who's being a persistent nuisance and draining whatever motivation I have for interacting with this project. Seriously. Do not push this any further.

Reply:
@deviant I do feel sorry for you and the situation. I do have to express though that I feel attacked by you. I think both, @SuperSandro2000 and you, are doing a great job by supporting the community.

Reply:
I am not rude as you describe it. I am just short and leave out the pretty fluw and explanations to get more reviews done.

I am also not harassing you or anyone else and I can't remember right now the last conversation with you if we had any at all. I am also not checking everybody's profile if they have blocked me because I don't really care. I am just looking at pull requests and adding notes about mistakes,  improvements and nits to improve the overhaul code quality.

I am also not having any problem when people jump in and continue the conversation by answering questions,  concerns or something else. I am not paid to do this and so it can happen that I take a few days to answer. I am also just a human which tend to forget things unlike computersand the internet.

PS: Next time please just answer that your name was taken on github and you would like to take V. That would have been most likely okay for me and we wouldn't have wasted all our time in a frankly not that productive discussion.

Reply:
> I am not rude as you describe it. I am just short and leave out the pretty fluw and explanations to get more reviews done.

I've had some of your reviews, and they are exactly as you advertise here. I appreciate _speakers_ being short and _listeners_ being kind. Not the other way round. After all, it's our all _time_ and _effectiveness_, that yo hold deer.

Reply:
Please act more civil. I don't think @SuperSandro2000 's original suggestion, or other review comments were unwarranted. "V" is not a very unique handle, and I would probably mistake it as a typo. If you're response was, "No, deviant was chosen because V wasn't available", I think the conversation would have ended there.

> Willfully ignoring being blocked by multiple people and choosing to continue to interact with them regardless is harassment by definition, not a "harsh" accusation. You didn't need to know, as we'd not be conversing presently had you not chosen to jump in earlier. If you think people haven't repeatedly tried to talk to him about his behaviour, you're sorely mistaken. There's no fight desired here, just someone lacking in social skills who's being a persistent nuisance and draining whatever motivation I have for interacting with this project. 

I feel like this unwarranted animosity toward Sandro. We should all be aligned with improving nixpkgs, and this and previous comments are not productive conversations.

> There's no fight desired here, just someone lacking in social skills who's being a persistent nuisance and draining whatever motivation I have for interacting with this project. 

At best this is highly dismissive, and at worst, verbal abuse. Please be accepting of other people's differences.

Reply:
> I've had some of your reviews, and they are exactly as you advertise here. I appreciate _speakers_ being short and _listeners_ being kind. Not the other way round. After all, it's our all _time_ and _effectiveness_, that you seem to hold deer.

Then it's fine for V to reply with a simple “no”, right?  They are being short, effective, and mindful of their time.

Reply:
> Then it's fine for V to reply with a simple “no”, right?

No.

---

<sub> ... it leaves you dangling :wink: </sub>

Reply:
> > Then it's fine for V to reply with a simple “no”, right?
> 
> No.

I was highlighting the double standard at work here but that seems to be too subtle an approach  >_>'

To be clear: ss2k's replies have the exact same problem; no justification is ever provided, so one is left to either guess or ask what's the actual issue and hope for a more-constructive answer.

Reply:
I also opined about listeners being _kind_. It's important, too.

From the whole context of this thread, I feel the [_good public faith_](https://en.wikipedia.org/wiki/Good_faith) is being under attack and that bothers me (deeply).

I do not understand on what (subjective) basis (any) elements of [malice](https://en.wikipedia.org/wiki/Mens_rea) are interpreted into the context and discussion.

Reply:
Names are not up for discussion.

Reply:
I am kind of dismayed by people seriously telling other people that they “can’t have a name”.

Reply:
> From the whole context of this thread, I feel the good faith is being under attack and that bothers me (deeply).
>
> I do not understand on what (subjective) basis (any) elements of malice are interpreted into the context and discussion.

Perhaps this is an indication that you are missing that context, and that you shouldn't be trying to judge interactions between two people who clearly have a history from the peanut gallery.

Reply:
@Profpatsch I did not try to tell V to not have a specific name. I just expressed my opinion about the nature of the conversation and about the name. However, I was very dismayed, to use your word, at the course of the discussion that followed. I should have expressed my opinion in a more appropriate way, but then, I can also expect to get a more proper, and less emotional, response.

Reply:
> I am kind of dismayed by people seriously telling other people that they “can’t have a name”.

Agreed but not sure if anyone actually said that. "Can you please use deviant?" is just a suggestion/question and due to https://github.com/deviant/nixpkgs/blob/0f01e4f1b959f49897f94c485e71069bd923cc19/maintainers/maintainer-list.nix#L27 that question makes IMO sense. But I can (more or less) understand all parties here and this is not me saying that `V` is bad or "wrong" but that, looking at this discussion, we might want to drop https://github.com/deviant/nixpkgs/blob/0f01e4f1b959f49897f94c485e71069bd923cc19/maintainers/maintainer-list.nix#L27 (not sure if this is also stated in other places) to avoid issues like this in the future.

Reply:
> I am kind of dismayed by people seriously telling other people that they “can’t have a name”.

I don't think that was the case at all, sandro suggested the github handle. Everyone agrees there's no technical reason to disallow a name like "V", just a subjective preference. Personally, I find it "easier" to mention someone to do `@<maintainer>` than `@$(nix eval .#lib.maintainers.<maintainer>.github)`, which is why personally I would prefer the github handle (Also, with a single letter, I'll probably have some additional cognitive dissonance about whether it's correct and have to verify). However, there's always the possibility of nixpkgs moving away from github, and trying to keep alignment with github would be irrelevant anyway.

Anyway, from a largely third party perspective, there's is discontinuity between the suggestion to use the github handle and everything that followed.